clar2wasm/src/standard/standard.wat

```diff
@@ -1409,29 +1409,81 @@
     )
 
     (func $stdlib.sha256-buf (param $offset i32) (param $length i32) (param $offset-result i32) (result i32 i32)
+        ;; For binary representation, you can take a look at https://sha256algorithm.com/
+        ;; For initial values, constants and high level understanding of SHA-512, please take a look at this Rust implementation
+        ;; https://github.com/dandyvica/sha/blob/master/src/sha256.rs
+        ;; If you're interested in the paper, please take a look at https://eprint.iacr.org/2010/548.pdf
```

Comment:
That's not the correct paper, this is the one for sha512/256

Reply:
Fixed

Reply:
No it's not, it's the same paper. Please don't resolve the conversation before the person who opened it had a chance to read the answer.

Reply:
Fixed